Chapters/1-0-Part1.md

```diff
@@ -0,0 +1,7 @@
+# Part I: Liffa
+
+_You cannot hear the first note of a ziyu's song. There is no grand entrance or catchy hook. No meandering conversation between a point and its counter. Each leaf is a gentle voice in a choir of the wind, finding its voice with the morning breeze. The tree is silent in the night, when the air is still and the stems curl up into themselves. But each morning, the light of Ara grows upon the horizon and awakens the breeze. You start to notice the sound, only it's been there for a while, and you can't place exactly when it began. By the time Ara is full, every leaf is perched and singing, their harmonies intertwined in unimaginable complexity._
+
```

Comment:
😮 you put some of it here... umm I guess okay.

The original prologue also gave us a very good sense of how Liffa saw the world, and it was nice.

I guess, I can see why you decided to include more action in the prologue.

Reply:
oh damn. I'll put some more action in the first chapter then.